accord-core/src/test/java/accord/burn/BurnTest.java

```diff
@@ -543,9 +544,11 @@ public static void main(String[] args)
         }
     }
 
-    @Test
+//    @Test
+    @RepeatedTest(Integer.MAX_VALUE)
```

Comment:
ill revert, just doing this to help run in the background and collect broken seeds